lib/datadog/core/telemetry/schemas/base/v1/application.rb

```diff
@@ -0,0 +1,31 @@
+module Datadog
+  module Core
+    module Telemetry
+      module Schemas
+        module Base
+          module V1
+            # Describes attributes for application environment object
+            class Application
+              attr_reader :language_name, :language_version, :service_name, :tracer_version, :env, :runtime_name,
```

Comment:
Minor, but from a style point of view, when you have multiple attributes, it's easier to read them as a vertical list, e.g.

```ruby
attr_reader \
  :foo,
  :bar,
  :baz
```